integration/simulation/l2_network.go

```diff
@@ -19,13 +19,30 @@ type L2NetworkCfg struct {
 	nodeAddresses    []string
```

Comment:
shouldn't the addresses be part of the p2p instance?
(In case of the "local", there are no addresses.)

Reply:
Not sure I follow. Locally all nodes have addresses, at the low level they're the different chans but we map them at a high level to a list of strings.